students/ChrisH/lesson02/grid_printer.py

```diff
@@ -0,0 +1,77 @@
+# ------------------------------------
+# grid_printer.py
+#  draw ascii grids
+# ------------------------------------
+
+P = '+'
+D = '-'
+V = '|'
+S = ' '
```

Comment:
avoid single character variable names that aren't throw aways used in for loops and such.